libnmstate/nm/team.py

```diff
@@ -63,12 +63,12 @@ def _convert_team_config_to_teamd_format(team_config, ifname):
     team_config = copy.deepcopy(team_config)
     team_config[TEAMD_JSON_DEVICE] = ifname
 
-    team_ports = team_config.get(Team.PORT_SUBTREE, ())
+    team_ports = team_config.pop(Team.PORT_SUBTREE, ())
```

Comment:
Isn’t this part of a different change? Looks like a fix, but I am unsure.

Reply:
No this is not part of a different change. We need to provide NM the teamd json configuration, on teamd the keyword is  `ports` so before we just needed to collect it and pass it. Now, we need to pop it out as using `port` is not correct on teamd json formatting.

I hope this answers your question, thanks!